leaderboard.js

```diff
@@ -130,19 +130,36 @@ function position(msg) {
 
     // Create new array with just names
     const userNames = Array.from(items, user => user[0]);
-  
-    // Find position of message author
-    const userPosition = userNames.indexOf(msg.author.username) + 1;
 
-    if (userPosition === 0) {
-      console.log("User is not on the leaderboard.");
+    const firstMention = msg.mentions.users.first();
+    if (firstMention) { // !== undefined && !== null
+      // Find position of message author
```

Comment:
Please correct this comment as this is finding the position of mentioned user

Reply:
Corrected: changed "message author" to "mentioned user" like you said to do.